commands/nextredline.rb

```diff
@@ -0,0 +1,48 @@
+require 'json'
+
+class Nextredline < BaseCommand
+  ## Note (This only responds with the first train south)
+  def perform
+    respond("From Kendall:
+            Next redline to ashmont comes at #{time_south_format}
+            Next redline to alewife from ashmont comes at #{time_north_format}")
+  end
+
+  def time_south_format
+    next_south_train_to_ashmont.strftime("%H:%M")
+  end
+
+  def time_north_format
+    next_train_north.strftime("%H:%M")
+  end
+
+  def next_train_north
+    if next_north_train_from_braintree < next_north_train_from_ashmont
+      next_north_train_from_braintree
+    else
+      next_north_train_from_ashmont
+    end
+  end
+
+  ## This is only the alewife -> ashmont line
+  def next_south_train_to_ashmont
+    Time.at(response_body["mode"][0]["route"][0]["direction"][0]["trip"][0]["pre_dt"].to_i)
```

Comment:
I like that you made the `response_body` and `response` methods. I'd make another for the route. Something like `ashmont_south`, `braintree_north`